build/app_template_files/__ReactNativeTemplateAppName__/__ReactNativeTemplateAppName__/AppDelegate.m

```diff
@@ -50,6 +50,8 @@ - (id)init
         [SalesforceSDKManager sharedManager].connectedAppId = RemoteAccessConsumerKey;
         [SalesforceSDKManager sharedManager].connectedAppCallbackUri = OAuthRedirectURI;
         [SalesforceSDKManager sharedManager].authScopes = @[ @"web", @"api" ];
+        [SalesforceSDKManager sharedManager].authenticateAtLaunch = NO;
```

Comment:
Actually, I will turn that into a comment - since we support both (automatic login at startup or deferred).
Also I will have a PR out shortly to support both on Android.